### PR TITLE
Use TS to connect to API instead of Python SDK

### PR DESCRIPTION
**Pull request by cpfarherFinitestate**

### CODE REVIEW

File: src/fs_main.ts [hunk 1]
```diff
@@ -0,0 +1,889 @@
...
+): Promise<axios.AxiosResponse> {
+  const response = await axios.put(url, bytes)
+
+  if (response.status === 200) {
+    return response
+  } else {
+    throw new Error(`Error: ${response.status} - ${response.statusText}`)
+  }
+}
+/*async function uploadBytesToUrl(url: string, chunk: Buffer): Promise<any> {
```
Comment (phillipcurl) [inline src/fs_main.ts:151]: Can we remove this?

File: src/fs_main.ts [hunk 2]
```diff
@@ -0,0 +1,889 @@
...
+    throw new Error('Artifact ID is required')
+  }
+  if (!testName) {
+    throw new Error('Test name is required')
+  }
+  if (!testType) {
+    throw new Error('Test type is required')
+  }
+
+  const graphqlQuery = `
```
Comment (phillipcurl) [inline src/fs_main.ts:352]: Do we want to move all of the queries and mutations in here to `./fs_queries.ts`?

**Review by phillipcurl — COMMENTED** [review]
Looking really good! Awesome work! A couple potential cleanup opportunities, but other than that, I think this should be good to go!

File: src/fs_main.ts (hunk 2 above)
Comment (cpfarherFinitestate) [reply src/fs_main.ts:352]: > Do we want to move all of the queries and mutations in here to `./fs_queries.ts`?

Do you mean move it to another file?

Comment (phillipcurl) [reply src/fs_main.ts:335]: Nope not at all!

**Review by phillipcurl — APPROVED** [review]

**Review by phillipcurl — APPROVED** [review]

**Pull request merged**